scootapi/client/run_job_cmd.go

```diff
@@ -33,13 +33,15 @@ func (c *runJobCmd) registerFlags() *cobra.Command {
 
 // Types to handle JobDefinitions from JSON files
 type JobDef struct {
-	Tasks map[string]TaskDef
+	Tasks     map[string]TaskDef
+	TimeoutMs int32
```

Comment:
Task timeout appears to flow through but job timeout is dropped in scheduler. We probably shouldn't add things unless we have a concrete timeline for using them.